recipes/platform.hashing/all/test_package/conanfile.py

```diff
@@ -1,13 +1,25 @@
 from conans import ConanFile, CMake, tools
+from conans.errors import ConanException
 import os
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake"
 
+    @property
+    def _extra_flags(self):
+        return self.deps_user_info["platform.hashing"].suggested_flags
+
     def build(self):
+        if self.settings.compiler != "Visual Studio":
+            if not self._extra_flags:
+                raise ConanException("Suggested flags are not available for os={}/arch={}".format(self.settings.os, self.settings.arch))
+
         cmake = CMake(self)
+        if self.settings.compiler != "Visual Studio":
+            cmake.definitions["EXTRA_FLAGS"] = self._extra_flags
```

Comment:
May use a more local name
Hmm. `LINKS_PLATFORM_EXTRA_FLAGS`

Reply:
It's the test package, so the name does not need to be namespaced.

Reply:
Ah. It's true.
Now you will have to use `conanfile.py` 😥

Reply:
Okay.
Now I will test it on my local machine